### PR TITLE
BLD: classifier using simple rasters

### PR DESCRIPTION
**Pull request by eigenfoo**

### CODE REVIEW

**Comment by eigenfoo** [comment]
Requesting review now, with the understanding that

1. the classifier will eventually ingest @Jped's data, instead of @ostapstephan's.
2. the classifier architecture is still not solidified.

File: src/parsettx.py [hunk 1]
```diff
@@ -3,13 +3,12 @@
...
-        'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z', 'zero', 'one', 'two',
-        'three', 'four', 'five', 'six', 'seven', 'eight', 'nine', 
-        'exclam', 'numbersign', 'dollar', 'percent', 'ampersand',
-        'asterisk','question', 'at' 
+    'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O',
+    'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z', 'a', 'b', 'c', 'd',
+    'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's',
+    't', 'u', 'v', 'w', 'x', 'y', 'z', 'zero', 'one', 'two', 'three', 'four',
+    'five', 'six', 'seven', 'eight', 'nine', 'exclam', 'numbersign', 'dollar',
+    'percent', 'ampersand', 'asterisk', 'question', 'at'
```
Comment (eigenfoo) [inline src/parsettx.py]: I'm not actually sure why this change is coming up in my PR (probably to do with the merge commit), but I'll just leave it here.

File: src/CoordConv.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        if self.with_r:
+            rr = torch.sqrt(
+                torch.pow(xx_channel.type_as(input_tensor) - 0.5, 2) +
+                torch.pow(yy_channel.type_as(input_tensor) - 0.5, 2))
+            ret = torch.cat([ret, rr], dim=1)
+
+        return ret
+
+
+class CoordConv(nn.Module):
```
Comment (ArianaFreitag) [inline src/CoordConv.py]: can you add some docs here, just bc I think this is a function that could be used in other contexts

Comment (eigenfoo) [reply src/CoordConv.py]: Added!

**Comment by eigenfoo** [comment]
@ArianaFreitag I'm okay with waiting until @Jped gets the new data: let's punt on this PR for now then.

**Comment by eigenfoo** [comment]
Superseded by #50

**Pull request closed without merging**